### ISSUE TITLE
Failed to parse JSON in select menu message 

### ISSUE THREAD

**Issue by loliGothicK** [serenity-rs/serenity#1437]
After selecting the Select Menu, it seems to fail to parse the JSON before `interaction_create` is called.

![image](https://user-images.githubusercontent.com/8962662/125158867-b0b2d080-e1ae-11eb-8462-4eb85f82482a.png)

Tracing output:
```
Jul 10 18:21:54.240  WARN run:recv_event:handle_event{event=Err(Json(Error("missing field `default`", line: 0, column: 0)))}: serenity::gateway::shard: [Shard [0, 1]] Unhandled error: Json(Error("missing field `default`", line: 0, column: 0))
```

**Comment by Zaitara** [comment]
Duplicate of #1428 which was fixed in #1418, use the latest commit or wait until a new version is released.

**Comment by loliGothicK** [comment]
Oh, I see.
